shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -356,6 +361,46 @@ - (BOOL)launchEngineInternalWithAssetsPath:(NSURL*)assets
   flutterArguments.command_line_argv = argv;
   flutterArguments.platform_message_callback = (FlutterPlatformMessageCallback)OnPlatformMessage;
 
+  // Provide data for release mode engine.
+#if NDEBUG
```

Comment:
@stuartmorgan Has the NDEBUG = AOT policy being settled upon already?

Reply:
Do you mean, have we settled on supporting AOT at all (vs. doing something like a symbol-stripped JIT)? Not specifically, but having some kind of release mode is clearly an improvement over not having one at all, and since it seems likely that AOT is what we will want (e.g., for startup time reasons), getting it in for people to try out lets see if there are issues, lets people try their own perf testing, etc.